misc/scripts/install-local.sh

```diff
@@ -44,17 +44,36 @@ function trap_ctrlc () {
 
 # run checks to verify script works
 function checks() {
+
+	if [[ -z "$name" ]]; then
+		fancy_message error "Package does not contain name"
+		exit 1
+	fi
+	if [[ -z "$hash" && "$name" != "*-git" ]]; then
+		fancy_message warn "Package does not contain a hash"
+	fi
+	if [[ -z "$description" ]]; then
```

Comment:
`description` is not necessary tbh

Reply:
oh wait did not notice you were not exiting. `maintainer` can be resolved but `description` should not show any warning as description is just for package maintainer's sanity